pvacseq/lib/main.py

```diff
@@ -31,7 +31,7 @@ def main(args_input = sys.argv[1:]):
     parser.add_argument("output_dir",
                         help="Output directory for writing all result files"
                         )
-    parser.add_argument("-l", "--variant-peptide-length",
+    parser.add_argument("-l", "--peptide-sequence-length",
```

Comment:
Huh, I didn't think that would work since the parameter names use dashes and the variables use underscore. Learned something new.